### PR TITLE
[V] Sorokyne Strata: Expanded Small corridors, removed 3000 unused tiles, Xenos can now weed everything except LZ's and surrounding of LZ's, allow LAZ for CAS inside structures with light blue Areas on Tacmap (glass ceiling hallways)

### PR DESCRIPTION
**Pull request by Venuska1117**
# About the pull request

This PR clean Sorokyne Strata file from excessive lines of code caused by APC's and small light fixture.
It changes some small rooms with have 1x1 hallways, now if possible, they are 2x2 (minimum).
Replaced 80% of indestructable walls with heavy reinforced walls, i dont see need for indestructable walls if they are not critical (for example, map edge)
Shortened river by half (south west near LZ) and placed invisible wall, i tough of preventing (in future) for burrowers to getting there and re-evolve to queen in spot where marines have no access because of invisible wall.
It "removes" walls with are 2tiles or more thick, i don't know why there are 2 tiles thick walls, but i believe they are not important.
Removes around 3000 unused tiles.

Edit: 08.08
- Allow xenos to weed everywhere (except LZ's)
- Allow marines to use LAZ for CAS inside most structures that have light blue colored areas (hallways made of glass ceiling)

Fixes: #6869

# Explain why it's good for the game

Less lines of code allow for quicker groundsite loading, even if its second faster, and also allow for more map destruction with less chockes caused by 1x1 tunnels, they don't appear much but they are "annoying", allow marines to use LAZ for CAS inside most structures (not all of them) and xenos can now weed 90% of areas except LZ's

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

img1

img2

img3

img4

img5

img6

img7

img8

img9

</details>

# Changelog

:cl: Venuska1117
maptweak: Sorokyne Strata - made most indestructable walls destructable.
maptweak: Sorokyne Strata - changed rooms/hallways with have 1x1 hallways to be wider.
maptweak: Sorokyne Strata - replaced some small light fixtures with normal light fixtures.
maptweak: Sorokyne Strata - reduced 2 tiles or more thick walls to 1 tile thick walls.
balance: Made most areas weedable, except LZ's zones and surrounding LZ's sub-variants.
balance: Allow LAZ inside Outposts (mining and normal ones) as long as they aim at light blue areas (glass ceiling hallways).
code: Changed pixel offset for "tube" lights to be closer to "floor".
/:cl:

### CODE REVIEW

File: maps/map_files/Sorokyne_Strata/Sorokyne_Strata.dmm
Comment (Nanu308) [inline maps/map_files/Sorokyne_Strata/Sorokyne_Strata.dmm:1]: image
Check if you can see down here with binocs/scopes, if so, add in proper walls if not it risks breaking immersion.

image
Door direction is wrong.


Other than that looking good 👍

**Review by Nanu308 — CHANGES_REQUESTED** [review]

**Comment by Venuska1117** [comment]
@Nanu308 can you please check map again, i removed changed south part of south jungle to remove around 3000-4000 unused tiles (considering i already removed long river) and moved some queen "nest" spots to better points (like one between hot springs and carp lake) more south of hot springs, i did this changes (movin nests) because by gameplay perspective you are unable to weed  water/coast.

File: maps/map_files/Sorokyne_Strata/Sorokyne_Strata.dmm
Comment (Nanu308) [inline maps/map_files/Sorokyne_Strata/Sorokyne_Strata.dmm:1]: image
image
image
Areas and stone walls, keep consistent

image
The room ends at the flooring, area here as well

image
Areas and floor tiles consistency. 

image
Door needs to be indestructible. 

image
image
Needs a light or two extra.


Go over the areas and the lighting, but good changes overall to the layout which should help a fair bit with the claustrophobic feeling of the map for the marines in particular.

**Review by Nanu308 — CHANGES_REQUESTED** [review]

File: maps/map_files/Sorokyne_Strata/Sorokyne_Strata.dmm
Comment (Venuska1117) [reply maps/map_files/Sorokyne_Strata/Sorokyne_Strata.dmm:1]: most of this area stuff was there by default, i didnt made those changes but thanks for checking them up, i will change them in next pull

Comment (Nanu308) [inline maps/map_files/Sorokyne_Strata/Sorokyne_Strata.dmm:1]: Need light/s:
image
image
image
image


Semiotic symbols, Check over computers, APC's, Restricted Areas such as armories, security for security areas, and so forth. Check the pinned message in the mapper channel for the general guide. For examples look at Almayer, it should be very up to par currently.

Some examples:
image
image
image
image


PR coming along nicely, put it in for testing 👍

**Review by Nanu308 — CHANGES_REQUESTED** [review]

File: maps/map_files/Sorokyne_Strata/Sorokyne_Strata.dmm
Comment (Venuska1117) [reply maps/map_files/Sorokyne_Strata/Sorokyne_Strata.dmm:1]: most of small light got removed and replaced with "long" lights, and to be honest, blub lights on glass windows looked strange (and they can be destroyed so sometimes it look like they float)

**Comment by Drulikar** [comment]
Stale exempt until Nanu makes a decision on this.

**Comment by Drulikar** [comment]
Be sure to replicate the change in https://github.com/cmss13-devs/cmss13/pull/6494 that you will likely conflict with.

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Review by Drulikar — CHANGES_REQUESTED** [review]
Its long overdue, but for https://github.com/cmss13-devs/cmss13/pull/6363 can you please expand the LZ or manipulate the areas (do so in code edits not var edits ideally) so that the LZ gas is how it would be done on other maps?

The gas is determined by whether the `is_landing_zone` var is set on the area. But note this also determines whether the area is mortar-able if lz protections are on. It also should not be too far from the dropship console because it is the dropship console that makes the fake OB to warn survivors they need to get out of there immediately.

You can test this in localhost by temporarily editing the `LZ_HAZARD_START` var in colonialmarines.dm and using the game panel (after load) to change the game mode to distress signal.

At the moment the gas looks like this on your version of the map (with my suggestions):
image
image

File: maps/map_files/Sorokyne_Strata/Sorokyne_Strata.dmm
Comment (Drulikar) [inline maps/map_files/Sorokyne_Strata/Sorokyne_Strata.dmm:1]: Needs to be expanded here (this area is reachable):
image

Consider expanding it here some:
image

Consider shrinking it here some:
image

**Review by Drulikar — DISMISSED** [review]

**Comment by Drulikar** [comment]
Gas looks fine now

**Comment by Git-Nivrak** [comment]
Most likely the cause for #6869

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

File: maps/map_files/Sorokyne_Strata/Sorokyne_Strata.dmm
Comment (Nanu308) [inline maps/map_files/Sorokyne_Strata/Sorokyne_Strata.dmm:1]: image
image
image
image
image
image
Pipes below walls/windows. Double check the system and route through areas that are open or under doors.

Cant spot anything else. Overall good improvement to the map. Can be merged after these fixes, might need further tweaks down the line.

**Review by Nanu308 — CHANGES_REQUESTED** [review]

File: maps/map_files/Sorokyne_Strata/Sorokyne_Strata.dmm
Comment (Venuska1117) [reply maps/map_files/Sorokyne_Strata/Sorokyne_Strata.dmm:1]: Changed pipes to follow standards: 
1. Dont go trough undestructable wall
2. Prioritize doors, if there is no access then window frames.
3. Make pipes as straight in line as possible, attach vents using closest path possible. 

(thats general rules I follow and applied it to your request, to make it look more logical.)

Comment (Nanu308) [inline maps/map_files/Sorokyne_Strata/Sorokyne_Strata.dmm:1]: Might be some oversights,  but nothing immediately popping out.

**Review by Nanu308 — APPROVED** [review]

File: maps/map_files/Sorokyne_Strata/Sorokyne_Strata.dmm
Comment (Venuska1117) [reply maps/map_files/Sorokyne_Strata/Sorokyne_Strata.dmm:1]: Great! Thank you!

**Pull request merged**
